syncer/syncer.go

```diff
@@ -616,12 +624,23 @@ func (s *Syncer) addJob(job *job) error {
 		s.jobWg.Wait()
 		addedJobsTotal.WithLabelValues("ddl", s.cfg.Name, adminQueueName).Inc()
 		s.jobWg.Add(1)
-		s.jobs[s.cfg.WorkerCount] <- job
+		queueBucket = s.cfg.WorkerCount
+		s.jobs[queueBucket] <- job
+		if job.ddlExecItem != nil {
+			execDDLReq = job.ddlExecItem.req
+		}
 	case insert, update, del:
 		s.jobWg.Add(1)
-		idx := int(utils.GenHashKey(job.key)) % s.cfg.WorkerCount
-		s.addCount(false, queueBucketMapping[idx], job.tp, 1)
-		s.jobs[idx] <- job
+		queueBucket = int(utils.GenHashKey(job.key)) % s.cfg.WorkerCount
+		s.addCount(false, s.queueBucketMapping[queueBucket], job.tp, 1)
+		s.jobs[queueBucket] <- job
+	}
+
+	if s.tracer.Enable() {
+		_, err := s.tracer.CollectSyncerJobEvent(job.traceID, job.traceGID, int32(job.tp), job.pos, job.currentPos, s.queueBucketMapping[queueBucket], job.sql, job.ddls, job.args, execDDLReq, pb.SyncerJobState_queued)
```

Comment:
I suggest we can write some detail in log （like sqls and args）， only store schema/ddl and so on

Reply:
and we can also output trace ID into log，then we can analyze them in production enviornment

Reply:
yep, I will refine this in another pr